RocketData/CollectionDataProvider.swift

```diff
@@ -156,7 +156,7 @@ open class CollectionDataProvider<T: SimpleModel>: ConsistencyManagerListener, B
      - parameter completion: Called on the main thread. This is called with the result from the cache.
      At this point, the data provider will already have new data, so there's no need to call setData.
     */
-    open func fetchDataFromCache(cacheKey: String?, context: Any? = nil, completion: @escaping ([T]?, NSError?)->()) {
+    open func fetchDataFromCache(withCacheKey cacheKey: String?, context: Any? = nil, completion: @escaping ([T]?, NSError?)->()) {
```

Comment:
Hmmmm...originally liked, but maybe not...I think I prefer cache key because it's consistent naming across different places?